api/envoy/extensions/watchdog/abort_action/v3alpha/abort_action.proto

```diff
@@ -17,7 +17,14 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Watchdog Action that sends a SIGABRT to kill the process.]
 // [#extension: envoy.watchdog.abort_action]
 
-// Configuration for the profile watchdog action.
+// A GuardDogAction that will terminate the process by sending SIGABRT to the
+// stuck thread. This would allow easier access to the call stack of the stuck
+// thread since we would run signal handlers on that thread. This would be
+// more useful than the default watchdog kill behaviors since those PANIC
+// from the watchdog's thread.
+
+// This is currently only implemented for systems that support kill to send
```

Comment:
Should this be the default, on supported platforms? Is there any downside to this? It seems to me that it gives better information (or a chance of it), and the same net behavior (process terminated). @envoyproxy/maintainers

Reply:
I'm fine with that being default in a future PR.

Reply:
But also fine here if we want this, seems generally useful.

Reply:
Sounds good, I can make it be a default action in a future PR.

The main downside I see of making it a default now would be different behaviors across platforms due to a lack of support on Windows. Should we wait for parity before we make it a default?

Reply:
I think it's ok to make it the default, because the only platform difference should be diagnostic output. The process is terminated on all platforms.

I'm also fine with making it the default in a future PR. It's your choice, @KBaichoo .

Reply:
Sounds good. I'll submit a follow up to this adding this action to the watchdog actions by default.

Reply:
+1 to make this the default.